examples/todos/android/src/main/java/org/reduxkotlin/example/todos/MainActivity.kt

```diff
@@ -0,0 +1,66 @@
+package org.reduxkotlin.example.todos
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import kotlinx.android.synthetic.main.activity_main.*
+import org.reduxkotlin.StoreSubscription
+import org.reduxkotlin.createStore
+import org.reduxkotlin.examples.todos.*
+
+/**
+ * This is a sample of basic redux behavior.
+ * This is NOT best practice for structuring a multiplatform App.
+ */
+
+
+val store = createStore(::rootReducer, AppState())
+
+class MainActivity: AppCompatActivity() {
+    private lateinit var storeSubscription: StoreSubscription
+    private var adapter = TodoAdapter()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        storeSubscription = store.subscribe { render(store.state) }
+        btn_add_todo.setOnClickListener {
```

Comment:
Since these view id's are being directly referenced in kotlin code, what do you think about camel casing them? That way the kotlin code will be more consistent style-wise

Reply:
Choose to keep snakecase to deliberately make views stand out as they are a special case.  I think real solution, and better practice going forward, is to use ViewBinding with camelcase.

Reply:
Updated with camelcase.  ViewBinding requires AS 3.6 so will wait on that to become stable